src/drive/web/modules/encryption/data.js

```diff
@@ -37,12 +36,13 @@ export const decryptData = async (key, data, { algorithm, iv } = {}) => {
   )
 }
 
-export const createDecryptedFileURL = async file => {
-  // prepare decryption
+export const createDecryptedFileURL = async (vaultKey, file) => {
+  // Prepare key + binary decryption
   const encryption = file.metadata.encryption
-  const iv = decodeArrayBuffer(encryption.iv)
-  // TODO the key should be encrypted
-  const key = await importKey('jwk', encryption.key)
+  const wrappedKey = decodeArrayBuffer(encryption.key.encrypted_key)
```

Comment:
Why a snake case for `encrypted_key` here ?

Reply:
See the `encrypted_key` param in https://tools.ietf.org/html/rfc7516#section-3.2

Reply:
My bad, you already told me once

Reply:
I forgive you because of your insights on the snake/camel cases